### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by corentin2l** [vertica/VerticaPy#1247] [Potential bug] [answered]
Hello,

We can't find the documentation of Verticapy on the Vertica website anymore.
The link https://www.vertica.com/python/documentation/ leads to https://www.opentext.com/products/vertica-data-platform

Where can we find the documentation for the 1.0.x version and the older versions ?

Regards,
Corentin

**Comment by oualib** [comment, marked as answer]
Hi Corentin,

Sorry for this issue out of my control. The website is again online!
